### PR TITLE
v1.3 release candidate

### PR DESCRIPTION
**Pull request by cpsievert**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,19 +1,19 @@
 Type: Package
 Package: sass
-Version: 0.1.2.9000
+Version: 0.1.3
```
Comment (jcheng5) [inline DESCRIPTION]: 0.2.0? Pretty significant new functionality in this release...?

File: DESCRIPTION [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 Authors@R: c(
-    person("Joe", "Cheng", , "joe@rstudio.com", c("aut", "cre")),
+    person("Joe", "Cheng", , "joe@rstudio.com", "aut"),
     person("Timothy", "Mastny", , "tim.mastny@gmail.com", "aut"),
     person("Richard", "Iannone", , "rich@rstudio.com", "aut",
        comment = c(ORCID = "0000-0003-3925-190X")),
     person("Barret", "Schloerke", , "barret@rstudio.com", "aut",
            comment = c(ORCID = "0000-0001-9986-114X")),
-    person("Carson", "Sievert", , "carson@rstudio.com", "aut",
+    person("Carson", "Sievert", , "carson@rstudio.com", c("aut", "cre"),
```
Comment (jcheng5) [inline DESCRIPTION]: We will definitely have to remember for me to send the transfer email when we submit

File: R/format.R [hunk 3]
```diff
@@ -36,6 +36,9 @@ knit_print.css <- function(x, options, ...) {
 }
 
 knit_print.sass <- function(x, options, ...) {
+  if (isTRUE(options$class.output %in% c('scss', 'sass'))) {
+    return(as.character(x))
+  }
```
Comment (jcheng5) [inline R/format.R]: Can you remind me what this is for?

Comment (jcheng5) [reply R/format.R]: Oh this is for your docs, basically?

Comment (cpsievert) [reply R/format.R]: Yea, it's analogous to what we already do for `knit_print.css()`:

https://github.com/rstudio/sass/pull/36/files#diff-2d8e8f1469f39259a828f967d4dc9cefR30-R33

**Comment by jcheng5** [comment]
LGTM (up to you whether to bump the minor version number)

**Pull request merged**